source/index.rst

```diff
@@ -1,44 +1,100 @@
 .. _index.proxysql-admin-interface:
 
-.. ProxySQL Admin Documentation documentation master file, created by
-   sphinx-quickstart on Fri Apr 17 21:29:13 2020.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
 
-|proxysql| and |proxysql-admin| documentation
+*ProxySQL*, **proxysql-admin**, and percona-scheduler documentation
```

Comment:
https://psqladm-373.onrender.com/index.html#:~:text=Edit%20this%20page-,ProxySQL%2C%20proxysql%2Dadmin%2C%20and%20percona%2Dscheduler%20documentation,%C2%B6,-ProxySQL%20is%20a

This makes all three product names to look differently. Is this expected?

Reply:
Fixed.